### DISCUSSION TITLE
Does Wave support JQuery as it does JS and HTML?

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#909] [Q&A] [answered]
Does Wave support JQuery as it does JS and HTML? Found a use case that is limiting in JS but can achieve with JQuery and was just wondering if this would compile as well. Thanks!

**Comment by racoss** [comment, marked as answer]
Check out this blog about the Wave JS integration:
https://wave.h2o.ai/blog/release-0.16

**Comment by lo5** [comment]
Yes, you can use JQuery (or any library for that matter). The technique is similar to the one described in the docs/blog.

**Reply by Alex-Wenner-FHR** [reply]
Thank you for the response!
